playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -17,6 +17,8 @@ parameter_defaults:
   NeutronDnsDomain: {{ clouddomain }}
   NeutronBridgeMappings: {{ neutron_bridge_mappings }}
   NeutronFlatNetworks: {{ neutron_flat_networks }}
+  NeutronNetworkType: 'flat,vlan'
```

Comment:
I think the default in TripleO with OVN is ['geneve', 'vxlan', 'vlan', 'flat'].
We should move that to ```playbooks/vars/defaults.yaml``` instead.